doc/Testing.md

```diff
@@ -13,3 +13,10 @@ Most grunt tasks will first launch a testing instance of the bot, to perform acc
 | `grunt wtest` | Watches tests for changes and runs tests | Yes |
 
 For examples on writing acceptance tests see [tests/acceptance/test_core_modules.js]
+
+
+### Testing modules
+
+Grunt will also run tests for modules installed that contain, test/acceptance and test/unit directories.
+
+For an example see the [aoi module](https://github.com/concierge/aoi)
```

Comment:
Full stop. Where possible I try to use the HelloConcierge module because that way it is more of a 'hello world' style example